vpr/src/route/rr_graph.cpp

```diff
@@ -106,8 +106,10 @@ static vtr::NdMatrix<std::vector<int>, 5> alloc_and_load_pin_to_track_map(const
                                                                           const int* sets_per_seg_type);
 
 static vtr::NdMatrix<int, 6> alloc_and_load_pin_to_seg_type(const e_pin_type pin_type,
```

Comment:
It would be good to add a doxygen comment saying what this routine does and describing the parameters.  If you don't know what all the parameters do, at least describing the new ones will be a step in the right direction.

Commenting what the Fc matrix stores: how many dimensions, what is in each dimension, and this is an absolute Fc value (I think).  Comment wht max_Fc is too (and the other parameters if you can).